base_ebill_payment_contract/tests/test_base_ebill_payment_contract.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from datetime import date, timedelta
+from odoo.tests.common import Form, SingleTransactionCase
```

Comment:
Nitpick: one line space between standard and third party imports.

Reply:
Done